common/app/implicits/FaciaContentFrontendHelpers.scala

```diff
@@ -28,23 +28,16 @@ object FaciaContentFrontendHelpers {
     def mainYouTubeMediaAtom: Option[MediaAtom] =
       for {
        main <- faciaContent.properties.maybeContent.map(_.fields.main)
-       atoms <-  faciaContent.properties.maybeContent.flatMap(_.atoms)
+       mediaAtoms <-  faciaContent.properties.maybeContent.map(_.elements.mediaAtoms)
        document <- Some(Jsoup.parse(main))
        atomContainer <- Option(document.getElementsByClass("element-atom").first())
        bodyElement <- Some(atomContainer.getElementsByTag("gu-atom"))
        atomId <- Some(bodyElement.attr("data-atom-id"))
-       mainMediaAtom <- atoms.media.find(ma => (ma.id == atomId && !ma.expired.getOrElse(false)) && ma.assets.exists(_.platform == MediaAssetPlatform.Youtube))
+       mainMediaAtom <- mediaAtoms.find(ma => (ma.id == atomId && !ma.expired.getOrElse(false)) && ma.assets.exists(_.platform == MediaAssetPlatform.Youtube))
```

Comment:
Just looked at it. This whole method is a workaround anyway. I the `TODO` mention, the correct way to achieve the same thing would be to use blocks.
Because of that I prefer not to touch this here since it is not the goal of this PR